apps/Linux Wifi Hotspot/uninstall

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#Allow packages required by this app to be uninstalled
+purge_packages || exit 1
+
+# Uninstall Linux Wifi Hotspot
+cd "$HOME"/linux-wifi-hotspot || error "Failed to cd into 'linux-wifi-hotspot'!"
```

Comment:
This will fail if the user first removes ~/linux-wifi-hotspot.
Also, this uninstall script does not remove ~/linux-wifi-hotspot.

A better solution would be to remove this folder during install, then make the uninstall script replicate whatever `sudo make uninstall` would do. This is easily done by looking removing all files mentioned in the folder's `installManifext.txt` file. (But please don't try to read that file on uninstall)

Reply:
there isn't an installmanifest (this app isn't quite a traditional cmake one)

it has hardcoded scripts
```
uninstall:
	rm -f $(DESTDIR)$(BINDIR)/create_ap
	rm -f $(DESTDIR)/etc/create_ap.conf
	rm -f $(DESTDIR)$(PREFIX)/lib/systemd/system/create_ap.service
	rm -f $(DESTDIR)$(PREFIX)/share/bash-completion/completions/create_ap
	rm -f $(DESTDIR)$(PREFIX)/share/doc/create_ap/README.md
	rm -f $(DESTDIR)$(BINDIR)/wihotspot
```

Reply:
I think I fixed it with [f51846f](https://github.com/Botspot/pi-apps/pull/1900/commits/f51846f59491e2b74418bcc2878fc86649ed167b).. Does this look good?